package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "liri",
+  "version": "1.0.0",
+  "description": "Calls twitter, spotify, or OMDB APIs",
+  "main": "index.js",
+  "scripts": {
```

Comment:
Let me know if you need help making scripts, it is more professional to just have scripts that can be ran.